test/tests/bcs/EquilibriumBC/tests

```diff
@@ -0,0 +1,10 @@
+[Tests]
+  [equilibriumBC_field_variable]
+    type = 'CSVDiff'
+    input = 'equilibriumBC.i'
+    csvdiff = 'equilibriumBC_out.csv'
+    issues = '#134'
+    design = 'EquilibriumBC.md'
+    requirement = 'The system shall compute the flux imposed by EquilibriumBC when a field variable is used for the enclosure variable.'
```

Comment:
```suggestion
    requirement = 'The system shall compute the equilibrium flux condition between an enclosure and a structure when a field variable is used for the enclosure variable.'
```

On second thought, let's not use the object name. The object and its usage should _ideally_ arise from the requirement, not the other way around.